tester_app/client/app/bundles/HelloWorld/store/helloWorldStore.jsx

```diff
@@ -0,0 +1,24 @@
+import { compose, createStore, applyMiddleware, combineReducers } from 'redux';
+import thunkMiddleware from 'redux-thunk';
```

Comment:
We should add a comment:
// See https://github.com/gaearon/redux-thunk
// This is not actually used for this simple example, but you'd probably want to use this once your app has
// asynchronous actions.